drivers/ads101x/ads101x.c

```diff
@@ -67,24 +65,35 @@ int ads101x_alert_init(ads101x_alert_t *dev,
     return _ads101x_init_test(I2C, ADDR);
 }
 
-int _ads101x_init_test(i2c_t i2c, uint8_t addr)
+static int _ads101x_init_test(i2c_t i2c, uint8_t addr)
 {
     uint8_t regs[2];
 
-    /* Acquire test */
     i2c_acquire(i2c);
-    if (i2c_init_master(i2c, I2C_SPEED) < 0) {
+
+    /* Register read test */
+    if (i2c_read_regs(i2c, addr, ADS101X_CONF_ADDR, &regs, 2, 0x0)
```

Comment:
I mostly saw a simple `0` for the flags in other places, IMHO we should stick to one style.

Reply:
I use 0 for defaults that are numerical, 0x0 for bit fields, and NULL for pointers. This is consistent with my prior work. IMO asking for developers to conform to a community standard for such detailed stylistic choices is burdensome without significant benefit.

Reply:
agreed, maybe the (most) correct way would be to use the `I2C_FLAG_NONE` to explicitly state that no flag should be set or used here. But we could leave that to a cleanup PR over all drivers and stuff later on.